string robot_id
string action_type
geometry_msgs/Pose initial_pose
geometry_msgs/Pose end_pose
string instructions
